### DISCUSSION TITLE
What would I need to edit to make it work with VLC instead?

### DISCUSSION THREAD

**Discussion by moeraccoon** [pystardust/ani-cli#130] [Q&A] [answered]
Assuming it's at all possible, I was wondering what I'd need to edit to make it work with VLC, it's my player of choice and I'd prefer that one.

**Comment by r4b61t** [comment]
try changing `player_fn="vlc"` in ani-cli ?

**Comment by port19x** [comment, marked as answer]
As of today: nothing
Just run with `ani-cli -v`
VLC is no officially and fully supported. 
If you run into trouble, open an issue
